### PR TITLE
Poster and report

### PR DESCRIPTION
**Pull request by DarthEkLen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,89 @@
-# DumE
-Open Projects 2021
+
+
```
Comment (theobscuredev) [inline README.md:2]: Delete these unnecessary line breaks throughout the README file.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,89 @@
-# DumE
-Open Projects 2021
+
+
+## DumE
+
+Open Projects 2021
+![DumE](https://github.com/rodion0917/Random/blob/main/Dum-E%20in%20Action%20-%20Green%20Object.png)
```
Comment (theobscuredev) [inline README.md:6]: Insert a line break after 'Open Projects 2021'. Also, center align all the images using the `<p>` paragraph tag and `align` attribute. For eg:
`<p align="center">`
  `LED Matrix`
  `<i>LED Matrix</i>`
`</p>`

And add appropriate title for each image.
Also try to resize the images by assigning proper dimensions.

Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

File: README.md [hunk 3]
```diff
@@ -1,2 +1,89 @@
...
+Our inspiration was simply to make something cool and learn the basics of robotics, computer vision,
+image processing and the various platforms which can be used for designing and simulating our robotic arm. 
+***
+
+
+
+## Workflow
+![Workflow](https://github.com/rodion0917/Random/blob/main/Work-flow%20chart.png)
+![Picking up the object - Workflow](https://github.com/rodion0917/Random/blob/main/Picking%20up%20the%20object%20-%20Workflow.png)
+![Placing the object - Workflow](https://github.com/rodion0917/Random/blob/main/Placing%20the%20object%20-%20Workflow.png)
```
Comment (theobscuredev) [inline README.md:32]: Again center align all the images and insert a title for each pic.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,89 @@
...
+***
+
+
```
Comment (theobscuredev) [inline README.md:35]: Insert a 'Software Aspect' heading and elaborate your control logic & algorithms here.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,89 @@
...
+
```
Comment (theobscuredev) [inline README.md:36]: Include a tabulated cost structure depicting the price of any paid software used. Note: MATLAB is free for IITR but not for other students. Make sure you mention it accordingly.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,89 @@
...
+Presently Dum-E can't function in a dynamic environment. Dum-E also has only a modest reach, which restricts it to lift only small, light objects.
+***
+
+
+## Future Improvements
+
+In future, we would work on making Dum-E function in a dynamic environment. Also, we try to enable Dum-E as a whole and not just its end-effector to move in dynamic environment. 
+***
+
+
```
Comment (theobscuredev) [inline README.md:55]: Remove these unnecessary line breaks.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,89 @@
...
+Ayush Mishra<br/>
+Shivshankar Shukla<br/>
+Siddhant Shekhar<br/>
+***
+
+
+
+## Mentors
+
+Rishika Chandra
```
Comment (theobscuredev) [inline README.md]: Hyperlink the Github ID of each team member & your mentor (https://github.com/chandrarishika14)

File: README.md [hunk 8]
```diff
@@ -1,2 +1,89 @@
...
+
+https://in.mathworks.com/help/robotics/ref/inversekinematics-system-object.html
+
+https://in.mathworks.com/help/robotics/ug/trace-end-effector-ik-simulink.html?searchHighlight=define%20the%20trajectory&s_tid=srchtitle
+
+https://in.mathworks.com/help/robotics/ug/2d-inverse-kinematics-example.html
+
+https://in.mathworks.com/help/robotics/ug/build-a-robot-step-by-step.html
+
+https://in.mathworks.com/help/robotics/ug/pick-and-place-workflow-using-stateflow.html
```
Comment (theobscuredev) [inline README.md:89]: Hyperlink the references properly. Mention the title for each reference and just include the URL in the hyperlink. Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

File: README.md [hunk 9]
```diff
@@ -1,2 +1,89 @@
...
+![DumE](https://github.com/rodion0917/Random/blob/main/Dum-E%20in%20Action%20-%20Green%20Object.png)
+***
+
+
+## Abstract
+
+Dum-E is a pick-up and place, robotic arm. Equipped with image processing and computer vision,it can 
+detect and classify objects on the basis of their color or shape. Currently, it can operate in static
+environments, that is where obstacles and the object are not moving.
+***
```
Comment (theobscuredev) [inline README.md:15]: Justify the entire text using the `<p>` paragraph tag and `justify` attribute. Make sure you justify all the paragraphs separately. For eg:
`<p align="justify">`
  `<h2>Abstract</h2>`
  `<p>A Rube Goldberg Machine is .........................</p>`
`</p>`

Refer this README for the same: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

**Review by theobscuredev — CHANGES_REQUESTED** [review]
Hey @DarthEkLen & team,

Nice work with the documentation although a few changes are required:

1. Update the README file according to the changes suggested below.
2. Update the name of the folders by referring the guidelines. Avoid custom names like 'Mechanical Model/ Robotic Arm'. Also update the name of 'Images and Videos' folder.
3. Change the format of your poster to .jpeg as it was mentioned in the guidelines. Also, report is missing from the 'Poster and Report' directory! 
4. Center align your mentor's name in the poster. Currently, it's top aligned.
5. Create only 2 folders namely 'Images' & 'Videos' inside the 'Images and Videos' folder and insert the appropriate images and videos in the respective directories.
6. Try to follow a uniform naming convention for each file. Either use camelCase (For eg: RoboticArm.sldasm or roboticArm.sldasm)/ use underscores (For eg: Robotic_Arm.sldasm)/ use hyphen (For eg: Robotic-Arm.sldasm), but ensure that it is uniform for EACH FILE IN YOUR REPOSITORY. Avoid naming the files as follows: arm-environment_workflow.png. Also ensure all the images uploaded have a same format (preferably .jpeg)
7. Remove any image/video from the 'Mechanical Model' folder (For eg: There's a Arm.JPG file).  They should only be present in 'Images and Videos' folder.

**Comment by theobscuredev** [comment]
@DarthEkLen Lemme know once you've made the final changes. Make sure to tag me (@) in this thread.

**Comment by DarthEkLen** [comment]
> @DarthEkLen Lemme know once you've made the final changes. Make sure to tag me (@) in this thread.

**Pull request closed without merging**